scripts/test-manual-e2e.sh

```diff
@@ -61,6 +61,8 @@ adb shell am start -n com.facebook.react.uiapp/.RNTesterActivity
 info "Press any key to open the workspace in Xcode, then build and test manually."
 info ""
 read -n 1
+success "Installing CocoaPods dependencies"
+rm -rf RNTester/Pods && cd RNTester && pod install
```

Comment:
Should this command also check that the pods repos are up to date?
```suggestion
rm -rf RNTester/Pods && cd RNTester && pod install  --repo-update
```

Reply:
IMHO, all dependencies are local and there is no need of updating repo, therefore it usually takes some time.